index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mansa Susa</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <header>
+      <h3 id="logo">Mansa Susa</h3>
+      <nav id="hamburger"><a href="#"><img src="imgs/Union.png" alt="hamburger"></a></nav>
+    </header>
+    <main>
+      <h1>Hey There.<br>I'm Pegasus</h1>
+      <article>
+        <h3>I'm a Software Engineer</h3>
+        <p>
+          I can help you build a product, feature or website. Look through some of
+          my work and experience!. If you like what you see and have a project you
+          need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] This font family is different from the original design for stick to the design please change it.

Reply:
The font family is the same as the original design which is 'Poppins' sans-serif is a fallback font family as it's the standard to always have a fallback font family in the case the primary fonts are not available.

Reply:
- Yes, you are right. You can ignore this one.